src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+import usersFromServer from '../../api/users';
+
+type Props = {
+  userId: number;
+};
+
+export const UserInfo:React.FC<Props> = ({ userId }) => {
+  // const { id, name, username, email }:User[] = usersFromServer;
+  const users:User[] = usersFromServer;
```

Comment:
[Checklist](https://github.com/mate-academy/react_add-todo-form/blob/master/newChecklist.md) №5. Prepare data in App component and pass it to child components.